qtconsole/console_widget.py

```diff
@@ -319,19 +333,22 @@ def __init__(self, parent=None, **kw):
         # Set a monospaced font.
         self.reset_font()
 
+        self.shortcut_manager = ShortcutManager()
+
         # Configure actions.
         action = QtWidgets.QAction('Print', None)
         action.setEnabled(True)
-        printkey = QtGui.QKeySequence(QtGui.QKeySequence.Print)
+        printkey = self.shortcut_manager.shortcut_print
```

Comment:
After applying locally the suggestions above an error appeared regarding the validation done below (using `QKeySequence.matches`). Due to that, here we need to keep `printkey` as a `QKeySequence` instance so this should look something like:
```suggestion
        printkey = QtGui.QKeySequence(self.shortcut_manager.shortcut_print)
```